src/core/core.helpers.js

```diff
@@ -669,6 +669,11 @@ module.exports = function(Chart) {
 	};
 	// Request animation polyfill - http://www.paulirish.com/2011/requestanimationframe-for-smart-animating/
 	helpers.requestAnimFrame = (function() {
+		if (typeof window === 'undefined') {
```

Comment:
I'm not sure how I feel about this change. I understand that it's rendering the chart synchronously if `window` is not defined, but that can be achieved by setting the animation duration to 0. @simonbrunel thoughts?